manifests/install.pp

```diff
@@ -199,8 +199,8 @@
     default: {
       case $facts['os']['family'] {
         'AIX': {
-          if "${python_version}" =~ /^python3/ { #lint:ignore:only_variable_string
-            class { 'python::pip::bootstap':
+          if String($::python::version) =~ /^python3/ {
```

Comment:
This and the String on L271 should not have :: prepends, or the uses above should have them - just for consistency.

Reply:
Adding prepends everywhere; I'm not sure which style is preferred.